source/tests/tf/test_data_large_batch.py

```diff
@@ -201,37 +201,37 @@ def test_data_mixed_type(self):
         np.savetxt("f.out", f.reshape([1, -1]), delimiter=",")
         np.savetxt("v.out", v.reshape([1, -1]), delimiter=",")
 
-        refe = [6.121172052273665543e01]
+        refe = [6.12116933882038480874e01]
```

Comment:
Is it a breaking change or just a random behavior change? When restarting from an old model, will anything be broken?

Reply:
This is a breaking change because the **layernorm** in TF (keras) is not consistent with those in PT/DP ( has numerical difference). It may be little confusing to keep two inplementations of Layernorm, should we keep old layernorm for TF? @njzjz @wanghan-iapcm

Reply:
It seems like a way to keep compatibility of TF while being consistent with DP/PT:
1. Add option for eps value in PT/DP, and they are consistent with TF when eps = 1e-3 (need test).
2. Remap the variable names to be compat with old models using keras.